definitions/ext-fastly_pop/definition.yml

```diff
@@ -1,21 +1,29 @@
-domain: EXT
 # Derived from the Fastly log ingestion, a datacenter is a Point of Presence
 # https://docs.fastly.com/en/guides/fastly-pop-locations
+domain: EXT
 type: FASTLY_POP
 
 synthesis:
-  name: fastly_datacenter
   identifier: fastly_datacenter
+  name: fastly_datacenter  
   encodeIdentifierInGUID: false
   conditions:
-#    The attribute’s value must match the provided value
-  - attribute: fastly_datacenter
-#      value: value
-
+    - attribute: fastly_datacenter
   tags:
     fastly_region:
+
+# Template that can be used to generate a dashboard for the entity.
```

Comment:
remove all this instead of commenting it please :D

Reply:
Done! :D